.cminx-config.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 CMakePP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+input:
+  # Whether undocumented functions should have documentation
+  # auto-generated.
+  include_undocumented_function: false
+
+  # Whether undocumented macros should have documentation
+  # auto-generated.
+  include_undocumented_macro: false
+
+  # Strips out any string matching this regex
+  # for parameter names. This is useful for when
+  # parameter names have a prefix/postfix to prevent
+  # name collisions. This affects function definitions.
+  #
+  # An example regex that strips the prefix "_abc_"
+  # from parameter names where "abc" is any combination of letters
+  # is below:
+  # function_parameter_name_strip_regex: "^_[a-zA-Z]*_"
+  function_parameter_name_strip_regex: "^_[a-zA-Z]+_"
```

Comment:
When using these regexes, we will need to update the `:param:` and other fields in the documentation

Reply:
I removed the prefixes and added the regex name stripping into the existing config at `docs/source/config.yml`.